public/components/tweets/tweets.controller.js

```diff
@@ -26,7 +26,7 @@ angular.module('textAnalysis')
 
             twitterService.loadTweets($routeParams.handle)
                 .then(function(data) {
-                    
+                    console.log(data);
```

Comment:
Don't commit console.log's, they can break production code for browsers that don't support console (older windows).

There are two options, one is to use console when testing only and remove them before pushing the code up.  Or, Angular has built in logging that you can use.  If you inject `$log` into your controller/service/etc then you can use it as a drop in replacement for console.

Example:

```
angular.module('myApp').controller('myCtrl', ['$scope', '$log', function($scope, $log) {
    $log.log('Some debugging information');
}]);
```